sw/tools/airframe_editor/paparazzi.py

```diff
@@ -1,81 +1,156 @@
 #!/usr/bin/env python
+# coding: utf-8
+
+r"""Paparazzi information retrieval"""
 
 from __future__ import print_function
 
 import glob
 
 from collections import namedtuple
-from os import path, getenv
-#from subprocess import call
-import commands
+from os import getenv
+from os.path import normpath, join, dirname, abspath
+# from subprocess import call
+# import commands
 
 import lxml.etree as ET
 
+import sys
+if sys.version_info[0] == 2:
+    PY2 = True
+else:
+    PY2 = False
+
+if PY2 is True:
+    from commands import getstatusoutput
+else:
+    from subprocess import getstatusoutput
+
```

Comment:
The if-structure and the boolean variable here are pretty redundant. Please use

```python
    PY2 = (sys.version_info[0] == 2)
    if PY2:
        #etc.
```
or, if `PY2` is never used again (as it seems to be here) consider directly inserting the condition expression into the last if-else-structure.